### PR TITLE
no module found on search

### LINKED ISSUE #286: When searching for deployed modules ,show
**Describe the bug**
When searching for deployed modules ,it shows blank if not found,
This would confuse the user thinking it has not been loaded.
he might think this is network issue
**To Reproduce**
1. Go to 'modules'
2. Click on 'search bar'
3. type a module that is not present 
4. See blank response

**Expected behavior**
show notfound card
**Screenshots**
240522_02h32m12s_screenshot

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by AKHIL-DyC**
closes #286 

## ✅ Checks
- [ ✅] I have updated the documentation as required
- [ ✅] I have performed a self-review of my code

### CODE REVIEW

**Comment by AKHIL-DyC** [comment]
this has changes of pr 285 also

**Comment by petar-cvit** [comment]
@AKHIL-DyC  I agree with the proposal, it should be clear that there were no modules found. Can we use [this](https://ant.design/components/empty) instead of the card?

**Comment by petar-cvit** [comment]
Can you center it in the middle of the screen? Also, I would remove the button since there already is one in the upper right corner and it might confuse.

Screenshot 2024-05-22 at 17 06 40

**Comment by AKHIL-DyC** [comment]
@petar-cvit  changes made

File: cyclops-ui/src/components/pages/Modules/Modules.tsx [hunk 1]
```diff
@@ -142,76 +143,82 @@ const Modules = () => {
...
-                      href={
-                        module.template.repo +
-                        `/tree/` +
-                        getTemplateVersion(module.template.version) +
-                        `/` +
-                        module.template.path
-                      }
+        {filteredData.length == 0 ? (
+          <div style={{ width: "100%" }}>
+            <Empty description="No Module Found"></Empty>
```
Comment (petar-cvit) [inline cyclops-ui/src/components/pages/Modules/Modules.tsx:148]:
```suggestion
            <Empty description="No Modules Found"></Empty>
```

**Review by petar-cvit — APPROVED** [review]
Thanks @AKHIL-DyC

**Pull request merged**